paywall/src/unlock.js/PostMessageEmitters/CheckoutIframeMessageEmitter.ts

```diff
@@ -59,13 +51,13 @@ export default class CheckoutIframeMessageEmitter extends FancyEmitter {
 
     this.buffer = []
 
-    const { postMessage, addHandler } = mainWindowPostOffice(
-      window,
-      this.iframe,
+    const { postMessage } = emitPostMessagesFrom(
+      this.iframe.contentWindow,
       url.origin,
-      'main window',
-      'Checkout UI iframe'
+      window,
```

Comment:
thank you for replacing the string with an object here!

Reply:
This is definitely one of those things where the old implementation did extra work to achieve less-good functionality.